### PR TITLE
change list name from answers_to_group to answers_to_calculate

### PR DESCRIPTION
**Pull request by shahi645**
Just a quick PR to change `answers_to_group` name to `answers_to_calculate`

### CODE REVIEW

File: schemas/questions/calculated.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
         "required": [
           "calculated_type",
           "answer_id",
-          "answers_to_group",
+          "answers_to_calculate",
```
Comment (jonnyshaw89) [inline schemas/questions/calculated.json]: I think the conditions values above should match our existing condition names
https://github.com/ONSdigital/eq-schema-validator/blob/master/schemas/common_definitions.json#L96

Comment (shahi645) [reply schemas/questions/calculated.json]: @jonnyshaw89 do u mean use the common definitions `conditions` or just the fact that they are a bit more readable i.e. `equal to`, `less than`, `less than or equal to`

Comment (jonnyshaw89) [reply schemas/questions/calculated.json]: just use the same values. it doesn't look good where we have multiple ways of writing the same thing

Comment (shahi645) [reply schemas/questions/calculated.json]: but they aren't actually the same thing, they happen to be named similarly, I actually prefer it with the hy[hens in there @nealedj second opinion ?

Comment (jonnyshaw89) [reply schemas/questions/calculated.json]: so `equals` isn't the same as `equal-to`?

Comment (dwyeradam) [reply schemas/questions/calculated.json]: I'm agreeing with Jon here. If **"equal-to"** is checking equality between two things then we have used **"equals"** across the schema. If **"equal-to"** is not checking the equality between two things then the naming is even worse.

Comment (dwyeradam) [reply schemas/questions/calculated.json]: Also we don't have a concept of **"less-than-equal-to"**. We would specify **"less than"** or **"equals"**.  Allowing multiple conditions to be specified so we have a greater range of possibilities but with less code.

File: schemas/questions/calculated.json [hunk 2]
```diff
@@ -28,73 +28,83 @@
...
-            },
-            {
-              "$ref": "../answers/number.json#/answer"
-            },
-            {
-              "$ref": "../answers/percentage.json#/answer"
+          "conditions": {
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 2,
```
Comment (jonnyshaw89) [inline schemas/questions/calculated.json]: Would it not be better to enforce unique than define the max?
`"uniqueItems": true,`

Comment (shahi645) [reply schemas/questions/calculated.json]: @jonnyshaw89 would we want both ?  otherwise with `uniqueItems` the user could enter 3 ?

Comment (jonnyshaw89) [reply schemas/questions/calculated.json]: yeh that is the case now you have added `greater than` before there were only 2 so you didn't need the max

**Comment by jonnyshaw89** [comment]
@shahi645 if you have a list in your commit message it should have a title on the 1st line

**Pull request merged**